.travis.yml

```diff
@@ -21,7 +21,7 @@ install:
 
 script:
   - make test
-  - make xlang
+  - make xlang-fresh
```

Comment:
I'm shocked that this is needed to somehow ensure the build runs freshly... is there some hidden cache somewhere in travis? Didn't think that was the case...